polyfill/modern.js

```diff
@@ -0,0 +1,11 @@
+/* eslint-disable @typescript-eslint/ban-ts-comment */
+
+// include all core-js/stable
+// webpack @babel/preset-env with useBuiltIns: 'entry' | 'usage'
+// will replace this with only the polyfills required at build-time.
+require('core-js/stable');
+
+// crypto is needed for PKCE
+require('fast-text-encoding'); // TextEncoder
+require('webcrypto-shim'); // crypto.subtle
+require('create-html-document-polyfill');
```

Comment:
We use a package called `html-react-parser` in `v3/src/hooks/useHtmlContentParser.tsx` which has sub-dependency that uses `document.implementation.createHtmlDocument` in a non-IE11 compatible way.

We can avoid this polyfill by using an old version of `html-react-parser` that uses an older sub-dependency that has IE11 compatibility code in it.

Both of these options are suboptimal:
1. The polyfill appears to be unmaintained, although this might be OK since it's simple and perhaps does not need any changing
2. An old version of `html-react-parser` would also be unmaintained and we would not be able to upgrade it for any security reasons stemming from it or subdependencies of it

One last thought I had was to have npm/yarn apply a patch on the package on postinstall since the issue can be solved with a 1 to 2 line change but I don't think that's a great solution either.